### ISSUE TITLE
Cannot connect an account which server requests adding certificate

### ISSUE THREAD

**Issue by Sof31t** [nylas/N1#344]
I am trying to add a new account with "IMAP / SMTP Setup". The setup is good but it just doesn't work, i am aware that this server has his own "unknown" security certificate and with other tools like thunderbird, i have to say that i trust this certificate to go on. The problem here it doesn't do the same stuff and it doesn't work...

**Comment by ecki** [comment]
I have the same problem with a startssl.com signed certificate. It worked to add the imap part, but when adding the smtp part it fails with no helpfull error "Invalid Credentials" after 10s.

Last time I tried I saw on my server closed STARTTLS attempts from EC2, but curretly there seems to be no connectionattempt at all? Does this maybe need some reset on the sync server?

(I will also file a request to allow to add IMAP only accounts)
